index.html

```diff
@@ -114,8 +104,9 @@ <h3>Terminology</h3>
+<p><dfn>Bidirectional text</dfn> is a term used to refer to text that mixes runs of both LTR and RTL text inline. It is common for text written in Arabic, Hebrew, etc. to contain short runs of Latin text, and several of the scripts that are written predominantly right-to-left write their numbers from left-to-right. Bidirectional text of this kind is the source of many of the difficulties when dealing with RTL scripts.</p>
 <p><dfn>Bidi isolation</dfn> often needs to be applied to a range of text in order to prevent the automatic rules of the Unicode Bidirectional Algorithm incorrectly ordering that content in relation to the surrounding text. For example, numbers following right-to-left text in memory are automatically positioned to the left of that text by the Bidi Algorithm, but sometimes need to appear to the right. Another example occurs when lists of RTL items occur in a LTR sentence: the Bidi Algorithm will automatically assume that the order of items in the list should be "3 ,2 ,1", but actually what's needed is "1, 2, 3". In HTML, bidi isolation can be applied to a range of text by enclosing it in an element with a <code class="kw" translate="no">dir</code> attribute. In plain text there are Unicode formatting characters that can do the job. These mechanisms remove unwanted 'spillover effects'.</p>
-<p> <dfn>First-strong detection</dfn> is an algorithm that looks for the first strongly-directional character in a string, and then uses that to guess at the appropriate base direction for the string as a whole. Unicode code points are associated with properties relating to text direction: generally, letters in right-to-left scripts such as Arabic and Hebrew have a strong RTL direction, whereas Latin and Han characters have a strong LTR direction. Other characters, such as punctuation, only have a weak intrinsic directionality, and the actual directionality is determined according to the context in which they are found.</p>
+<p> <dfn>First-strong detection</dfn> is an algorithm that looks for the first strongly-directional character in a string that is not inside a nested isolate, and then uses that to guess at the appropriate base direction for the string as a whole. Unicode code points are associated with properties relating to text direction: generally, letters in right-to-left scripts such as Arabic and Hebrew have a strong RTL direction, whereas Latin and Han characters have a strong LTR direction. Other characters, such as punctuation, only have a weak intrinsic directionality, and the actual directionality is determined according to the context in which they are found.</p>
```

Comment:
_that is not inside a nested isolate_ is kind of mysterious here. I know what you mean, but it took me a bit to grok it. Perhaps:

**First-strong detection** is an algorithm that looks for the first strongly-directional character in a string (while ignoring isolates), and then uses that...